client/core/core.go

```diff
@@ -1687,6 +1855,12 @@ var respHandlers = map[string]routeHandler{
 func (c *Core) listen(dc *dexConnection) {
 	defer c.wg.Done()
 	msgs := dc.MessageSource()
+	// Lets run a match check every 1/3 broadcast timeout.
+	//
+	// DRAFT NOTE: This is set as seconds in server/dex/dex.go, but I think it
+	// should be milliseconds, no?
+	bTimeout := time.Second * time.Duration(dc.cfg.BroadcastTimeout)
+	ticker := time.NewTicker(bTimeout / 3)
```

Comment:
This runs a check for action once every 1/3 * broadcast timeout. In theory, this check shouldn't be necessary, so I might want to remove it.